packages/medusa-plugin-sendgrid/src/services/sendgrid.js

```diff
@@ -1155,6 +1163,30 @@ class SendGridService extends NotificationService {
     return data
   }
 
+  async orderRefundCreatedData({ id, refund_id }) {
+    const order = await this.orderService_.retrieveWithTotals(id, {
+      select: [
+        'total',
+      ],
+      relations: [
+        'refunds',
+        'items',
```

Comment:
todo: Use double quotes
```suggestion
        "total",
      ],
      relations: [
        "refunds",
        "items",
```

Reply:
@olivermrbl Sweet! I just cleaned these few things up that you mentioned here.

Yeah I have tested it locally and everything was working for me.